bytelatent/data/iterators/arrow_iterator.py

```diff
@@ -236,7 +236,7 @@ def create_iter(
 
     def _set_row_num(self, target_row_num: int):
         logger.info(
-            f"Setting arrow position to {target_row_num} for {self.dataset_files}"
+            f"Setting arrow position to {target_row_num} for {str(self.dataset_files)[:200]}"
```

Comment:
Perhaps mention in the print that its the top 200 in this case

Reply:
Just add a .... or something to indicate that we arent printing the entire list, so that we dont look at the log and wrongly infer that some datasets are being skipped